src/Form/form-autosuggest.mdx

```diff
@@ -22,21 +22,25 @@ Form auto-suggest enables users to manually select or type to find matching opti
     const [selected, setSelected] = useState('');
 
     return (
-        <Form.Autosuggest
-            floatingLabel="Programming language"
-            aria-label="form autosuggest"
-            helpMessage="Select language"
-            errorMessageText="Error, no selected value"
-            value={selected}
-            onSelected={(value) => setSelected(value)}
-        >
-            <Form.AutosuggestOption>JavaScript</Form.AutosuggestOption>
-            <Form.AutosuggestOption>Python</Form.AutosuggestOption>
-            <Form.AutosuggestOption>Rube</Form.AutosuggestOption>
-            <Form.AutosuggestOption onClick={(e) => alert(e.currentTarget.getAttribute('data-value'))}>
-                Option with custom onClick
-            </Form.AutosuggestOption>
-        </Form.Autosuggest>
+        <Form.Group>
```

Comment:
It seems to me that this `mdx` file should have 2 indent spaces (currently 4). This can be seen by looking at other `mdx` files of the [Form component](https://github.dev/openedx/paragon/blob/716a6ddd932d61a5b5ab594e8e525c3dfffe20ab/src/Form/form-autosuggest.mdx).

Reply:
Thanks! I updated all the jsx in `from-autosuggest.mdx` https://github.com/openedx/paragon/compare/ea7a1a6fde65eac7566541d5d8be3a5b7af526dd..a1f5abbcae9703bdb7bd1e80f5f093a84d7aa36b